contributing.md

```diff
@@ -35,7 +35,9 @@ Regarding Pybind, Here is a good [introduction](https://www.youtube.com/watch?v=
 To contribute to PyDP you will need to fork the OpenMind/PyDP repository.
 Then you can work risk-free on your fork.
 
-You will just need to fork once. After that you can call `git fetch upstream` and `git pull 'branch-name'` before you do your local changes to get the remote changes and be up-to-date
+You will just need to fork once. After that you can call `git fetch upstream` and `git pull 'branch-name'` before you do your local changes to get the remote changes and be up-to-date.
```

Comment:
Can we please make it simpler?

Reply:
Call it dev?

Reply:
Yeah, could be called that. Simpler and per norms.